runway/blueprints/staticsite/dependencies.py

```diff
@@ -110,17 +115,30 @@ def create_template(self):
             Value=artifacts.ref()
         ))
 
+        callbacks = self.context.hook_data['aae_callback_url_retriever']['callback_urls']
+
         if variables['AuthAtEdge']:
+            user_pool_id = variables['UserPoolId']
+
+            if variables['CreateUserPool']:
+                user_pool = template.add_resource(
+                    cognito.UserPool("AuthAtEdgeUserPool")
```

Comment:
(entirely style & preference) can we make the resource just "UserPool" -- there won't be any other user pools in the stack & its nice to avoid prefixing all the resources with the general stack concept.

Reply:
Should we do the same with the Client? It'd take a little refactoring but not much.

Reply:
That is interesting -- we can't update the client now (stacks are already deployed with that resourceid).

I'm not sure it's a big deal to have them be inconsistent, but it's fine leaving this as AuthAtEdgeUserPool if it seems more consistent.

Reply:
Ok, I'll leave as is just to keep the naming consistent.